ci/run.sh

```diff
@@ -95,12 +96,14 @@ fi
 
 case "$TARGET" in
   *-apple-ios)
-    cargo rustc --manifest-path libc-test/Cargo.toml --target $TARGET -- \
-        -C link-args=-mios-simulator-version-min=7.0
+    cargo rustc --manifest-path libc-test/Cargo.toml --target $TARGET \
+        --test main -- -C link-args=-mios-simulator-version-min=7.0
+    cargo rustc --manifest-path libc-test/Cargo.toml --target $TARGET \
+        --test linux-fcntl -- -C link-args=-mios-simulator-version-min=7.0
```

Comment:
I've fixed this on the latest build, but it failed on i386-ios because there were two `main-*` binaries. Which is weird, but maybe it's a caching issue. I'm not too certain what the caching policy is when set to `cargo` on Travis, but maybe that'll need to be turned off? Any other ideas here?